### PR TITLE
Update wiiu.md

### PR DESCRIPTION
**Pull request by ExperiencersInternational**
Fixed incorrect spelling of navigate, and fixed PNID not being capitalised on the last line, as well as adding a full stop to that sentence.

### CODE REVIEW

**Comment by ExperiencersInternational** [comment]
I did a pretty bad job at noticing some of the other minor mistakes oops

File: docs/en_US/install/wiiu.md [hunk 1]
```diff
@@ -90,11 +90,15 @@ You will want to register an account from [here](/account) and click `Don't have
...
+Right before registering a PNID, please make sure to save your password on some sort of note, as you will not be able to reset your password. You will want to go on you Wii U while Pretendo is still active and press the top left profile icon. After pressing the icon, press `Switch Account`, and then `Add New User`. Set up this account as normal, but once it asks if you have an account, press `No` and go through the account process normally. When it asks you again to link after adding a Mii, press `Link`. Choose your language and then accept the Pretendo Network Service Agreement.
 
 <img src="/assets/images/docs/install/wiiu/pnid/pretendo-eula.jpg" width="100%">
 
-Finally, set up your pnid, email and password. Once all of this is done you will be able to use Pretendo Server
+Finally, set up your PNID, email and password. Once all of this is done, you will be able to use Pretendo Servers.
```
Comment (jonbarrow) [inline docs/en_US/install/wiiu.md:104]:
```suggestion
Finally, set up your PNID, email and password. Once all of this is done, you will be able to use Pretendo servers.
```

File: docs/en_US/install/wiiu.md [hunk 2]
```diff
@@ -90,11 +90,15 @@ You will want to register an account from [here](/account) and click `Don't have
...
-Right before registering a PNID, please make sure to save your password on some sort of note, as you will not be able to reset your password. After registering just simply change your birth date, gender, timezone, country/region, etc to what you see fit. Once you have it set up we can switch back to the Wii U. Within your Wii U you will want to reconnect to Pretendo _**Note:** how you reconnect to Pretendo depends on how you installed it on your Wii U!_ Once you are reconnected, you will want to press the top left profile icon and press `Switch Account`, then press `Add New User`. Go through the user set up normally, but when it asks if you have an account, press `Yes` and put in your username and password. If it asks you to confirm your email afterwards, simply skip it. You will now be able to use Pretendo Servers
+Right before registering a PNID, please make sure to save your password on some sort of note, as you will not be able to reset your password. After registering just simply change your birth date, gender, timezone, country/region, etc to what you see fit. Once you have it set up we can switch back to the Wii U. Within your Wii U you will want to reconnect to Pretendo.
```
Comment (jonbarrow) [inline docs/en_US/install/wiiu.md:93]: While we're here, password resetting has been implemented for a long time. So the section about it not being implemented is no longer needed

**Review by jonbarrow — CHANGES_REQUESTED** [review]

**Comment by ExperiencersInternational** [comment]
Should be resolved now, let me know if there are still issues. Might edit the 3DS guide as well considering it doesn't tell you how to sign in on a 3DS (just did this), and it might be useful just telling people how to go to Luma settings if need be

I need to get my Wii U modded already in all honesty

**Comment by gitlimes** [comment]
Looks good to me! @jonbarrow hbu?

**Comment by jonbarrow** [comment]
@gitlimes i trust your judgment, you can merge if it looks good

**Comment by gitlimes** [comment]
Well, merging then. Thank you for your contribution!

**Pull request merged**